app/services/problem_check/force_https.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ProblemCheck::ForceHttps < ProblemCheck
+  self.priority = "low"
+
+  def call
+    return no_problem if SiteSetting.force_https
```

Comment:
I think this should be `if !SiteSetting.force_https`?

Reply:
Actually this is one of the most confusing ones. I had my brain twisted multiple times.

The logic of the check is this:

- If a request comes in over HTTPS, check that `force_https` is `true`.

So if `force_https` is `true`, there's no problem.

Reply:
Ahhh OK cool :+1: